index/scorch/segment/zap/docvalues.go

```diff
@@ -35,6 +35,16 @@ type docValueIterator struct {
 	curChunkData   []byte // compressed data cache
 }
 
+func (di *docValueIterator) sizeInBytes() uint64 {
+	// curChunkNum, numChunks, dvDataLoc
+	sizeInBytes := 24
+	sizeInBytes += len(di.field)
+	sizeInBytes += len(di.chunkLens) * 8
+	sizeInBytes += len(di.curChunkData)
```

Comment:
since curChunkData is initialized from s.mm which is mmap'd, then we shouldn't count this in the calculation which is intended to count space on the heap